blackbox/docs/sql/aggregation.txt

```diff
@@ -205,8 +205,13 @@ Some Examples::
 =======
 
 The ``sum`` aggregation function returns the sum of all the values in a column
-that are not ``NULL`` as a double value. Its single argument is the column name
-of a primitive numeric column. ``sum`` can not be used on other column types.
+that are not ``NULL``. Depending on the argument type a suitable return type
+is chosen. For float and double argument types the return type is equal to the
+argument type. For byte, short, integer and long the return type changes
+to long. In case of exceeding the range of long values (-2^64 to 2^64-1) an
+ArithmeticException will raise.
```

Comment:
replace: If the range of long values (-2^64 to 2^64-1) gets exceeded an ArithmeticException will be raised.